Makefile

```diff
@@ -25,9 +23,28 @@ $(PYTHON):
 	$(VIRTUALENV) $(VENV)
 	$(VENV)/bin/pip install -U pip
 
+build-requirements:
+	@rm -fr /tmp/xml2kinto
```

Comment:
you are making the assumption /tmp exists. a better thing to do is to grab the tmp env variable make provides